apis/network/v1/gkenetworkparamset_types.go

```diff
@@ -126,6 +126,10 @@ const (
 	DPDKUnsupported GKENetworkParamSetConditionReason = "DPDKUnsupported"
 	// GNPReady indicates that this GNP resource has been successfully validated and Ready=True
 	GNPReady GKENetworkParamSetConditionReason = "GNPReady"
+	// NetworkAttachmentInvalid indicates that the specified network attachment is invalid.
+	NetworkAttachmentInvalid GKENetworkParamSetConditionReason = "NetworkAttachmentInvalid"
+	// GNPConfigInvalidArgument indicates that the GNP config has an invalid combination of fields specified.
+	GNPConfigInvalidArgument GKENetworkParamSetConditionReason = "GNPConfigInvalidArgument"
```

Comment:
When invalid combination of fields is specified, I believe the entire config is valid rather than just an argument. I think `GNPConfigInvalid` better conveys that and we could trim `Argument` from it

Reply:
Makes sense -- updating name